frontend/src/global_styles/content/_table.sass

```diff
@@ -216,10 +216,6 @@ table.generic-table
       &.-no-highlighting
         background-color: $body-background
 
-    .op-uc-p
-      @include text-shortener
```

Comment:
I think you might be breaking other stuff by removing this. Could just an override for CF be enough?

Reply:
The `td` already has the text-shortener mixin unless explicitly disabled, so this should be safe to remove